src/onegov/fsi/assets/js/ifs.js

```diff
@@ -0,0 +1,26 @@
+function autoResizebyID(id) {
+    var newheight;
+    var newwidth;
+
+    if (document.getElementById) {
```

Comment:
Not really necessary, since this function has been supported by browsers going back as far as IE 5.5. Also, if for some reason this condition is false, an error will occur further down anyway.